src/rpc/core_rpc_server.cpp

```diff
@@ -91,6 +92,8 @@ namespace cryptonote
     command_line::add_arg(desc, arg_restricted_rpc);
     command_line::add_arg(desc, arg_rpc_ssl);
     command_line::add_arg(desc, arg_rpc_ssl_private_key);
+    command_line::add_arg(desc, arg_rpc_ssl_private_key_passphrase);
+    command_line::add_arg(desc, arg_rpc_ssl_private_key_passphrase_file);
```

Comment:
As per @omartijn comments, I don't see a reason to support/maintain a passphrase file specifically in the core rpc code. I don't see a good reason for the command line-switch passphrase either, although people aggressively running in pid namespaces (i.e. containers) can hide the argument list in some cases. It seems like a `stdin` or prompt would be the only useful scenario.